app/models/concerns/id.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
-module Paginable
```

Comment:
After some thinking, I am convinced with an idea that `ID` is a valid name conforming the convention to name concerns after related properties.
Namely, `AccountAvatar`, `AccountHeader`, and `AccountInteractions` are representing properties those concerns deal with. `ID` concern also deals with IDs.